lib/utils.js

```diff
@@ -34,7 +34,8 @@ function makeRtpEngineOpts(req, srcIsUsingSrtp, dstIsUsingSrtp, teams = false) {
     'call-id': req.get('Call-ID'),
     'replace': ['origin', 'session-connection'],
     'record call': process.env.JAMBONES_RECORD_ALL_CALLS ? 'yes' : 'no',
-    ...(process.env.JAMBONES_ACCEPT_G729 && { codec: { mask: 'g729', transcode: 'pcmu' } })
+    ...(process.env.JAMBONES_ACCEPT_G729 && { codec: { mask: 'g729', transcode: 'pcmu' } }),
```

Comment:
how about both JAMBONES_ACCEPT_G729 and JAMBONES_OPUS_PCMU_TRANSCODE enabled ????

Reply:
I think we still need to support JAMBONES_ACCEPT_G729 for backwards compatibility.  However, we should implement also a more general solution which is a list of codecs to accept, but to transcode to PCMU/PCMA if received, e.g. "JAMBONES_ACCEPT_AND_TRANSCODE: "opus, g722"

Reply:
@davehorton just for discussion like genesys once can choose codecs setting at carrier end , this would be more generic feature and specific to carrier based on customer needs

Reply:
fixed, tested on jambonz.me